lib/nerves_hub/devices/alarms.ex

```diff
@@ -61,6 +62,19 @@ defmodule NervesHub.Devices.Alarms do
     |> Repo.one!()
   end
 
+  def get_current_alarms_for_device(device) do
+    device.id
+    |> Devices.get_latest_health()
+    |> case do
+      %DeviceHealth{data: %{"alarms" => alarms}} when is_map(alarms) ->
+        for {alarm, description} <- alarms,
+            do: {String.trim_leading(alarm, "Elixir."), description}
```

Comment:
```suggestion
        for {alarm, description} <- alarms, into: %{},
            do: {String.trim_leading(alarm, "Elixir."), description}
```
I don't know if it matters. I originally thought these were a list, but looks like they are a map. If that's a case, then this would ensure it's placed back into a map

Reply:
Yes, the alarms comes in a map. For current use cases it's fine to pass them on as maps here.